op-node/withdrawals/utils.go

```diff
@@ -255,14 +263,54 @@ func ParseWithdrawalInitiated(receipt *types.Receipt) (*bindings.L2ToL1MessagePa
 	if err != nil {
 		return nil, err
 	}
-	if len(receipt.Logs) != 1 {
-		return nil, errors.New("invalid length of logs")
+	abi, err := bindings.L2ToL1MessagePasserMetaData.GetAbi()
+	if err != nil {
+		return nil, err
 	}
-	ev, err := contract.ParseWithdrawalInitiated(*receipt.Logs[0])
+
```

Comment:
This will allow the withdrawal to come from a contract that emits events and still be parsed, altho it won't work on multiple withdrawals in the same tx. It will always parse the first one. I think this is fine for now